Lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4490,20 +4494,29 @@ namespace Js
 
         if (!fOnlyCurrent)
         {
-            for (uint uIndex = 0; uIndex < this->m_nestedCount; uIndex++)
+            this->ForEachNestedFunc([&](FunctionProxy* proxy, uint32 index)
             {
-                Js::ParseableFunctionInfo * pBody = this->GetNestedFunctionForExecution(uIndex);
+                FunctionProxy* currentNestedFunction = this->GetNestedFunc(index);
```

Comment:
using the ForEach so it does not need to get the array everytime, just notice this is debugger only code path, let me change it back